app/src/main/java/protect/card_locker/ShortcutHelper.java

```diff
@@ -155,4 +155,26 @@ Bitmap createAdaptiveBitmap(@NotNull Bitmap in, int paddingColor) {
                 .setIntent(intent)
                 .setIcon(icon);
     }
+    static ShortcutInfoCompat.Builder createGroupShortcutBuilder(Context context, Group group) {
+        Intent intent = new Intent(context, MainActivity.class);
+        intent.setAction(Intent.ACTION_MAIN);
+        // Prevent instances of the view activity from piling up; if one exists let this
+        // one replace it.
+        intent.setFlags(intent.getFlags() | Intent.FLAG_ACTIVITY_SINGLE_TOP);
+        final Bundle bundle = new Bundle();
+        // Done like with the cards but with group._id (a string)
+        bundle.putString("id", group._id);
+        bundle.putBoolean("view", true);
```

Comment:
This boolean seems unused?

Reply:
Yes I think you are right. I noticed it might also be the case with "createShortcutBuilder" (for the cards shortcuts) just above it?